src/libxrpl/protocol/STAmount.cpp

```diff
@@ -1419,10 +1420,28 @@ mulRoundImpl(
         return STAmount(v1.getFName(), minV * maxV);
     }
 
+    if (v1.holds<MPTIssue>() && v2.holds<MPTIssue>() && asset.holds<MPTIssue>())
+    {
+        std::uint64_t minV = (getMPTValue(v1) < getMPTValue(v2))
+            ? getMPTValue(v1)
+            : getMPTValue(v2);
+        std::uint64_t maxV = (getMPTValue(v1) < getMPTValue(v2))
+            ? getMPTValue(v2)
+            : getMPTValue(v1);
```

Comment:
As above, is there a reason you can't just use `min` and `max`?
```suggestion
        std::uint64_t const minV = std::min(getMPTValue(v1), getMPTValue(v2));
        std::uint64_t const maxV = std::max(getMPTValue(v1), getMPTValue(v2));
```